TailorSort.java

```diff
@@ -43,7 +44,32 @@ private static boolean isArraySorted(Integer[] a){
         //WRITE YOUR CODE HERE
         //RETURN THE RESULT COATS ARRAY WITH ALL SORTED
         //YOU HAVE TO DO MINIMUM COMPARISONS, CLUE: USE index parameter in your sorting technique
+    	Integer key ;
+    	int swapIndex = 0;
+     	for (int i = 0; i < inputCoats.length ;i++ ){
```

Comment:
I asked in icebreaker group " In TailorSort, Math.random is used. It gives value from 0-1 and hence the unsorted array has number less than sorted array. Do we need to sort the whole array or change the Math.random()*35 to (Math.random()+35) * 10) so that is always greater than 35?" You replied to sort the whole array.